api/serverV2.go

```diff
@@ -65,7 +66,8 @@ func NewServerV2(
 
 	wrappedWeb3Handler := otelhttp.NewHandler(newHTTPHandler(web3Handler), "web3.jsonrpc")
 
-	wrappedWebsocketHandler := otelhttp.NewHandler(NewWebsocketHandler(web3Handler), "web3.websocket")
+	limiter := rate.NewLimiter(rate.Limit(cfg.WebsocketMaxRateMessages), 1)
```

Comment:
Does it means allowing 5 req/s and only 1 req at the same time?

Reply:
> Does it means allowing 5 req/s and only 1 req at the same time?


once a client is connected, the WebSocket server will limit the request rate to 5 req/s for this client. There is no limit to the number of client sides connected to the server( can limit in gateway)

Reply:
so, what does the second param `1` means?

Reply:
> so, what does the second param `1` means?

The  second parameter determines the maximum burst events that the limiter can tolerate. For example, if the value of `b` is 10, then even if 10 events suddenly occur in a short time. set to `1` means that the rate limit is based on the setting of the first parameter, and too many bursts are not allowed.